### PR TITLE
Default vars for ckan jekyll

### PR DESCRIPTION
**Pull request by FuhuXia**
add defaults for catalog, inventory, and jekyll playbooks

### CODE REVIEW

**Review by vasili4 — APPROVED** [review]

File: ansible/roles/software/ckan/saml2/defaults/main.yml [hunk 1]
```diff
@@ -15,3 +15,7 @@ saml2_pkgs:
       repo: "https://github.com/GSA/pysaml2",
       branch: "max"
     }
+
+# provide some default values to be overwritten later by group vars and host vars...
+project_source_new_code_path: /usr/lib/ckan-new
+who_ini_secret: e45cfed3-40f1-41c0-8e92-77eda7ddd9f3
```
Comment (JJediny) [inline ansible/roles/software/ckan/saml2/defaults/main.yml:21]: Confirmed this is a default random key

Comment (JJediny) [reply ansible/roles/software/ckan/saml2/defaults/main.yml:21]: OK

File: ansible/roles/software/inventory/ckan-app/defaults/main.yml [hunk 2]
```diff
@@ -39,3 +39,21 @@ datapusher_pkgs:
       repo: "https://github.com/GSA/datapusher",
       requirements: "requirements.txt"
     }
+
+# provide some default values to be overwritten later by group vars and host vars...
+ckan_instance_secret: a97df940-4ac1-4357-aac2-156d4f21d300
```
Comment (JJediny) [inline ansible/roles/software/inventory/ckan-app/defaults/main.yml:44]: Random key that gets overwritten by secrets OK

**Review by JJediny — APPROVED** [review]
Review completed - OK

**Comment by JJediny** [comment]
Review completed

**Pull request merged**